src/python/pants/backend/python/dependency_inference/BUILD

```diff
@@ -1,11 +1,7 @@
 # Copyright 2020 Pants project contributors (see CONTRIBUTORS.md).
 # Licensed under the Apache License, Version 2.0 (see LICENSE).
 
-python_sources(
-    dependencies=[
-        "./scripts:dependency_parser",
```

Comment:
asset dep inference handles this fine, so no need to be explicit

Reply:
This was a showcase of asset inference :smile: